docs/user_guide.md

```diff
@@ -17,18 +17,18 @@ cd Simoorg
 ```
 When installing from the source, we recommend that you run all the unit and integration tests that are included with the source code, before starting the installations. You can do this by running the command
 ```
-    python setuptools.py test
+    sudo python setup.py test
```

Comment:
Hi @MayureshGharat 

IF i do $ python setuptools.py test

[ec2-user@ip- simoorg]$ ls
docs  failure_scripts  LICENSE  MANIFEST.in  NOTICE  README.rst  requirements.txt  sample_configs  setup.cfg  setup.py  src
[ec2-user@ip- simoorg]$ python setuptools.py test
python: can't open file 'setuptools.py': [Errno 2] No such file or directory
[ec2-user@ip- simoorg]$

However if I DO:

[ec2-user@ip simoorg]$ sudo python setup.py test
running test
running egg_info
creating src/simoorg.egg-info
writing requirements to src/simoorg.egg-info/requires.txt
writing src/simoorg.egg-info/PKG-INFO
writing top-level names to src/simoorg.egg-info/top_level.txt
writing dependency_links to src/simoorg.egg-info/dependency_links.txt
writing entry points to src/simoorg.egg-info/entry_points.txt
writing manifest file 'src/simoorg.egg-info/SOURCES.txt'
reading manifest file 'src/simoorg.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'product-spec.json'
warning: no files found matching 'mppy-pinned.txt'
writing manifest file 'src/simoorg.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
[ec2-user@ip simoorg]$